README.md

```diff
@@ -4,37 +4,180 @@
+TestCafe will compile your test code on-flight and run it immediately.
+It also ships with built-in support for source maps to leverage debugging experience.
+Source maps are automatically enabled, so all you need to do is start a debugging session in an IDE that supports them.
+
+TestCafe introduces simple, but powerful test API. It offers a couple of dozen methods covering all user actions.
+Chained syntax produces code that is easy to write and read.
+Furthermore, you are free to use any assertion library to perform verifications of different kinds.
+
+```js
+import { expect } from 'chai';
+
+fixture `My Fixture`
+    .page('http://devexpress.github.io/testcafe/example');
+
+test('My Test', async t => {
+    await t
+        .click('#send-button')
+        .handleAlert()
+        .typeText('#input', 'Peter Parker')
+        .wait(1000);
+
+    expect(await t.eval(() => getSomethingOnTheClient())).to.be.true;
+});
+```
+
+## Write Regular Client JavaScript to Observe Page State
+
+TestCafe introduces special kinds of functions that power you to observe the page state as if you are inside the page - `Selector` and `ClientFunction`.
+Simply write JavaScript code within these functions, it will be executed on the client side, giving you direct access to DOM elements or obtaining some data from the client.
+These functions are called from a test as a regular async function, so you can pass parameters to it and get the function result.
+
+```js
+import { Selector } from 'testcafe';
+
+const getElementById = Selector(id => document.querySelector(`#${id}`));
+
+fixture `My Fixture`
+    .page('http://devexpress.github.io/testcafe/example');
+
+test('My Test', async t => {
+    await t
+        .typeText('#developer-name', 'John Smith')
+        .click('#submit-button');
+
+    const articleHeader = await getElementById('article-header');
+});
+```
+
+## Descriptive Reports
+
+TestCafe automatically generates full-detailed reports providing a test run summary and comprehensive information about errors.
+Fancy call sites, clean stacks and screenshots help you easily detect an error cause.
+
+Choose from five built-in reporters to output test results or create custom reporter plugins to produce your own reports.
+
+![Spec Reporter](docs/articles/images/spec-reporter.png)
+
+# Documentation
+
+- [Getting Started](http://devexpress.github.io/testcafe/documentation/getting-started/)
+- [Test API](http://devexpress.github.io/testcafe/documentation/test-api/)
+- [Using TestCafe](http://devexpress.github.io/testcafe/documentation/using-testcafe/)
+- [Extending TestCafe](http://devexpress.github.io/testcafe/documentation/extending-testcafe/)
+- [Recipes](http://devexpress.github.io/testcafe/documentation/recipes/)
+
+# Contributing
+
+Please use our issues page to report a bug or request a feature.
+
+For more information on how to help us improve TestCafe, please see the Contributing.md file.
```

Comment:
It will be created in context of https://github.com/DevExpress/testcafe/issues/703 I guess